ts/Core/Pointer.ts

```diff
@@ -212,15 +211,24 @@ class Pointer {
             if (series.navigatorSeries) {
                 activeSeries.push(series.navigatorSeries);
             }
+
+            // Include boosed series when they share markerGroup
+            if (series.boosted && series.markerGroup) {
+                activeSeries = activeSeries.concat(
+                    this.chart.series.filter(function (otherSeries): boolean {
+                        return otherSeries.markerGroup === series.markerGroup;
+                    })
```

Comment:
Arrow function is terser. Maybe also use `push.apply` instead of concat?
```suggestion
                    this.chart.series.filter((otherSeries): boolean =>
                        otherSeries.markerGroup === series.markerGroup
                    )
```